File: 0268-missing-number/0268-missing-number.cpp
class Solution {
public:
    int missingNumber(vector<int>& nums) {
        int cnt=0;
        int flag=0;
        for(int i=0;i<=nums.size();i++){
            for(int j=0;j<nums.size();j++){
                if(i==nums[j]){
                    flag=1;
                    cnt++;
                    break;
                }
                else{flag=0;}
            }
            if(flag==0){
                    break;
                }
        }
        return cnt;
    }
};

File: 0268-missing-number/0268-missing-number.java
class Solution {
    public int missingNumber(int[] a) {
        int n=a.length;
        int sum1=(n*(n+1))/2;
        int sum2=0;
        for(int x:a){
            sum2+=x;
        }
        return sum1-sum2;
    }
}